ODIN_II/SRC/include/odin_types.h

```diff
@@ -101,8 +104,8 @@ struct global_args_t
     argparse::ArgValue<bool> all_warnings;
     argparse::ArgValue<bool> show_help;
 
-	argparse::ArgValue<std::string> adder_def; //DEPRECATED
-
+	argparse::ArgValue<std::string> adder_def; //carry skip adder skip size
```

Comment:
don't use it this way. this expects either default, optimized or a file name. Make a new one and use a boolean to enable GA adder